### PR TITLE
added circleci fixes #10

### LINKED ISSUE #10: Add circleci
Once we have our testing and linting setup we can setup our circle builds.  There's some steps on setting it up down at the bottom of this:

https://circleci.com/blog/continuously-testing-react-applications-with-jest-and-enzyme/

### PR DESCRIPTION
**Pull request by zyjenny**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/redux-async
```
Comment (jaredkerim) [inline .circleci/config.yml]: This should probably be something like 'avocado' or something?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    docker:
+      - image: circleci/node:8
+    steps:
+      - checkout
+      - restore_cache:
+          key: npm-cache-v1-{{ checksum "package-lock.json" }}
```
Comment (jaredkerim) [inline .circleci/config.yml]: Very nice 👍

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      - run:
+          name: Install Dependencies
+          command: npm ci
+      - save_cache:
+          key: npm-cache-v1-{{ checksum "package-lock.json" }}
+          paths:
+            - /home/circleci/.npm
+      - run:
```
Comment (jaredkerim) [inline .circleci/config.yml]: Yep, and add a second one for linting.

File: .circleci/config.yml (hunk 1 above)
Comment (zyjenny) [reply .circleci/config.yml]: changed 🥑

File: .circleci/config.yml (hunk 3 above)
Comment (zyjenny) [reply .circleci/config.yml]: Screen Shot 2019-05-24 at 3 30 13 PM

is this the right thing to add?

**Pull request merged**